.github/workflows/tests.yml

```diff
@@ -4,6 +4,13 @@ on:
     branches:
       - develop
   pull_request:
+    paths:
```

Comment:
what is `paths` for?

Reply:
It's used to specify a list of files that should trigger the workflow when changes are made to them, so that a commit that changes say changelog only won't rerun tests.